ts/features/wallet/component/features/PaymentMethodInitiatives.tsx

```diff
@@ -61,7 +55,7 @@ const PaymentMethodInitiatives = (props: Props): React.ReactElement | null => {
   return initiativesList.length > 0 ? (
     <View testID="idPayInitiativesList" style={props.style}>
       <View style={[IOStyles.rowSpaceBetween, IOStyles.alignCenter]}>
-        <SectionHeader iconName="initiatives" title="wallet.capability.title" />
+        <H3>{I18n.t("wallet.capability.title")}</H3>
```

Comment:
```suggestion
        <H6>{I18n.t("wallet.capability.title")}</H6>
```

Reply:
should all components in that page use H6? this would be the outcome
<img width="368" alt="Screenshot 2023-08-07 at 12 41 20" src="https://github.com/pagopa/io-app/assets/60693085/093d6060-8827-401a-94d7-7d6ac4def0d4">

Reply:
> should all components in that page use H6? this would be the outcome <img alt="Screenshot 2023-08-07 at 12 41 20" width="368" src="https://user-images.githubusercontent.com/60693085/258778156-093d6060-8827-401a-94d7-7d6ac4def0d4.png">

Yes, as per design on Figma the section headers should be H6. If the scope of this PR is to replace the top component, we should leave the old `SectionHeader`, but I don't see any reason to use a H3.

Reply:
the scope is to overall apply the new design, so I believe H6 does fill the request.
that said, I also updated to H6 for the settings and initiatives listItems, if it does look right to you I can push so you can approve/re-review this PR

Reply:
Yes, list items are H6 but black instead of grey-700 👌🏻

Reply:
addressed 👍

Reply:
Sorry, didn't notice it was a global component! We should not touch it for the moment.